src/proxyvisor.coffee

```diff
@@ -137,7 +137,7 @@ createProxyvisorRouter = (proxyvisor) ->
 		.then ([ device ]) ->
 			return res.status(404).send('Device not found') if !device?
 			return res.status(410).send('Device deleted') if device.markedForDeletion
-			proxyvisor.logger.log(m, { channel: "device-#{device.logs_channel}-logs" })
+			proxyvisor.logger.logDependent(m, { channel: "device-#{device.logs_channel}-logs" })
```

Comment:
This call will need to pass the device uuid in the second parameter else the log call will fail for native logging

Reply:
Ah, good catch, I lost this in the rebase